src/diem/testing/miniwallet/app/static/openapi.yaml

```diff
@@ -84,7 +79,7 @@ paths:
           $ref: '#/components/responses/ClientError'
   /accounts/{account_id}/balances:
     get:
-      summary: Get account balances
+      summary: This method gets the currency balances for the account specified.
```

Comment:
summary has to be short, it becomes manual line item on the left tab

so `Get account balances` is better